src/applications/check-in/pre-check-in/pages/Error/index.jsx

```diff
@@ -56,21 +57,28 @@ const Error = () => {
     </>
   );
 
-  const getErrorMessage = () => {
+  const getErrorMessages = () => {
     if (appointments && appointments.length) {
       // don't show sub message if we are 15 minutes past appointment start time
-      if (appointmentStartTimePast15(appointments)) return '';
+      if (appointmentStartTimePast15(appointments)) return ['', null];
```

Comment:
Do we know for sure whether the accordions should not be present on the "past appointment start time" page as well?

Reply:
That is a good question, the ticket title was specific to "Expired" but I'll dig around to see if there is another ticket and look at the mocks.

Reply:
From my understanding, anytime after midnight day of is expired, including 15+ min after appointment start time. Which to me means both situations should have the accordions. But I may have misunderstood what "expired" means!

Reply:
I think you are right. Here is the sketch: https://www.sketch.com/s/5331b114-280d-4ff5-8d36-ec49b1696b9e/p/5D647D7E-68C0-4E3B-948C-162CE11A9EB0